images/base/Dockerfile

```diff
@@ -70,6 +70,7 @@ RUN echo "Ensuring scripts are executable ..." \
       libseccomp2 pigz \
       bash ca-certificates curl rsync \
       nfs-common \
+      jq \
```

Comment:
how big is this?

Reply:
about 1MB including deps
https://packages.ubuntu.com/focal/jq

Reply:
thanks, paying 1MB for rootless seems worthwhile :-)